tests/rados/pool_tests.py

```diff
@@ -565,6 +565,178 @@ def run(ceph_cluster, **kw):
         rados_obj.detete_pool(pool=pool_name)
         return 0
 
+    if config.get("Verify_degraded_pool_min_size_behaviour"):
+        log.info(
+            "Test to verify that degraded Pools are not able to serve clients below min_size"
+        )
+        pool_target_configs = config["Verify_degraded_pool_min_size_behaviour"][
+            "pool_config"
+        ]
+        pool_name = ""
+        pgid = ""
+        osd_pick = ""
+        second_osd_pick = ""
+        for entry in pool_target_configs.values():
+            pool_name = entry["pool_name"]
+            try:
+                method_should_succeed(rados_obj.create_pool, **entry)
+                # Write IO to pool
+                rados_obj.bench_write(pool_name=pool_name)
+
+                log.info("Getting a sample PG from the pool")
+                pool_id = pool_obj.get_pool_id(pool_name=pool_name)
+                pgid = f"{pool_id}.0"
+                pg_set = rados_obj.get_pg_acting_set(pg_num=pgid)
+                log.info(
+                    f"Pg set fetched from pool id {pool_id} for pg id {pgid} is {pg_set}"
+                )
+                osd_tree_op = rados_obj.run_ceph_command(cmd="ceph osd tree")
+                log.info(
+                    f"Before bringing down osds, output from ceph osd tree : {osd_tree_op}"
+                )
+
+                osd_pick = pg_set.pop()
+                second_osd_pick = pg_set.pop()
+                log.info(
+                    f"Bringing down two OSDs from the acting set : {osd_pick} and {second_osd_pick}"
+                )
+                log.info(f"The OSD that will be remaining in the pg_set is : {pg_set}")
+                if not rados_obj.change_osd_state(
+                    action="stop", target=osd_pick, timeout=20
+                ):
+                    log.error(f"Failed to stop OSD : {osd_pick}")
+                    raise Exception("test-bed set-up failure")
+
+                if not rados_obj.change_osd_state(
+                    action="stop", target=second_osd_pick, timeout=20
+                ):
+                    log.error(f"Failed to stop OSD : {osd_pick}")
```

Comment:
Very minor - Please change `{osd_pick}` to `{second_osd_pick}`

Reply:
done